requirements.txt

```diff
@@ -11,7 +11,7 @@ coinpaprika>=0.1.0
 requests>=2.28.1
 safe-cli==0.7.0
 safe-eth-py==5.5.0
-slackclient==2.9.4
+slackclient
```

Comment:
Maybe we can pin whichever version it is you wound up getting with this?

You could check with

```sh
pip freeze | grep slack
```